src/components/shared/form_elements/RadioFormInput.vue

```diff
@@ -1,45 +1,46 @@
 <template>
-	<label
-		ref="labelRef"
+	<div
 		class="radio radio-form-input"
-		:class="{ 'is-disabled': disabled, 'active': inputModel === nativeValue }"
-		@click="focus"
-		@keydown.prevent.enter="click"
```

Comment:
Removing this event handler will lead to form submit (and subsequent popping up of error messages for all unfilled fields). With the current "jump to first error" functionality removed, I see this as a step backwards in usability. But I don't see this as a blocker.

Is there a general recommendation on how the Enter key should behave, what action it should trigger in a form and/or form field? If it's generally agreed upon that enter should submit the form, then let's remove this event handler.